js/scripts.js

```diff
@@ -1,22 +1,39 @@
-var url = 'http://api.icndb.com/jokes/random';
+var prefix = "https://cors-anywhere.herokuapp.com/";
+var tweetLink = "https://twitter.com/intent/tweet?text=";
+var quoteUrl = "https://quotesondesign.com/wp-json/posts?filter[orderby]=rand&filter[posts_per_page]=1";
 
-var button = document.getElementById('get-joke');
+function getQuote() {
+    $.getJSON(quoteUrl, createTweet);
+    $.getJSON(prefix + quoteUrl, createTweet);
+    $.ajaxSetup({ cache: false });
+}
 
-button.addEventListener('click', function() {
-  getJoke();
-});
+function createTweet(input) {
+    var data = input[0];
```

Comment:
before you take first item from array you should be sure that your variable is an array, so before you assign to `data` variable some value, check that `input` is an array and contains at least one element